rakelib/support/vic_load_test.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module LoadTest
+  module_function
+
+  def measure_elapsed
+    t0 = Time.zone.now
+    yield
+    t1 = Time.zone.now
+    elapsed = t1 - t0
+
+    puts
+    puts "Start:      #{t0}"
+    puts "End:        #{t1}"
+    puts "Elapsed:    #{elapsed.round(2)} seconds"
+  end
+
+  def conn(route)
+    Faraday.new("http://localhost:3000/v0/vic/#{route}") do |c|
```

Comment:
We want to exercise the complete path through the revproxies/ELBs so we want to be able to run this from an external host against https://staging-api.vets.gov. If you can parameterize this via a rake task parameter that would be good.

Reply:
Yep, no problem